### DISCUSSION TITLE
Polling impact on 12 V battery

### DISCUSSION THREAD

**Discussion by Dave-Casey** [dahlb/ha_kia_hyundai#116] [General]
In the community of Kia Hyundai electric vehicle owners, a consistent drain on 12V auxiliary batteries has been linked to the use of home assistant software for remote vehicle monitoring. This software, designed to frequently query the car for status updates, inadvertently prevents the vehicle from entering its low-power rest state. The high polling rate of such systems results in continuous activation of the car’s electronics, which are dependent on the 12V battery, leading to significant energy depletion when the vehicle is not in use.

To address this issue, it is recommended that EV owners adjust the polling settings of their home assistant applications to reduce the frequency of data requests. This measure has been reported by the owner community to successfully preserve the 12V battery’s charge. Maintaining a balance between the convenience of real-time monitoring and the vehicle’s energy conservation is crucial for preventing such unintended battery discharge.

**Comment by dahlb** [comment]
<img width="403" alt="Screenshot 2023-12-29 at 8 27 33 PM" src="https://github.com/dahlb/ha_kia_hyundai/assets/54117/0405bf17-4318-4660-8b04-1ae0c257c0b5">

the default for this integration is to only request fresh car data every 4 hours, do you have any references for the 12v drain? 

I wouldn't be shocked if kia did a bad design causing occasional data requests to cause 12v battery issues, but I'd hope it needs more frequent polling then every 4 hours.

**Comment by dahlb** [comment]
This is no longer a concern since the v1.9.0 rewrite https://github.com/dahlb/ha_kia_hyundai/releases/tag/v1.9.0 which removes all automated forced requests to wake up the vehicle
